src/client/core/utils/content-editable.js

```diff
@@ -137,6 +138,10 @@ export function getFirstVisibleTextNode (el) {
     return child;
 }
 
+export function getFirstVisibleTextNode (el) {
```

Comment:
I think we don't need this function anymore

Reply:
This function is used in several places.
I think it's more convenient to write `getFirstVisibleTextNode()` instead of `getFirstTextNode(true)`